### PR TITLE
Update Traditional Chinese (zh_tw.json)

### PR DESCRIPTION
**Pull request by dirtTW**
Update and Fix Traditional Chinese localization

### CODE REVIEW

File: src/main/resources/assets/jade/lang/zh_tw.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	"tooltip.jade.villager_profession": "職業：%s",
 	"gui.jade.by": "Snownee 用%s製作",
 	"gui.jade.by.hovered": "查看我的所有作品！",
-	"gui.jade.translated_by": "%s 繁體中文本地化",
+	"gui.jade.translated_by": "%s 繁體中文在地化",
 	"gui.jade.translators": "Jimmy-sheep, notlin4, HJ-zhtw, dirtTW",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/zh_tw.json:20]: please add your nickname here

File: src/main/resources/assets/jade/lang/zh_tw.json [hunk 2]
```diff
@@ -95,6 +95,7 @@
 	"config.jade.display_item_hide": "隱藏",
 	"config.jade.display_item_top": "頂部",
 	"config.jade.display_item_centered": "置中",
+	"config.jade.display_item_inline": "內嵌",
 	"config.jade.tts_mode": "朗讀功能模式",
 	"config.jade.tts_mode_extra_msg": "朗讀功能,朗讀,按下,切換",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/zh_tw.json]: could you help me remove the words in all "extra_msg" entries that already appear in the title? (朗讀功能,朗讀)

thanks!

**Review by Snownee — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/jade/lang/zh_tw.json (hunk 1 above)
Comment (dirtTW) [reply src/main/resources/assets/jade/lang/zh_tw.json:20]: I am dirtTW

File: src/main/resources/assets/jade/lang/zh_tw.json (hunk 2 above)
Comment (dirtTW) [reply src/main/resources/assets/jade/lang/zh_tw.json]: Sure!

**Pull request merged**